charts/opentelemetry-collector/templates/_helpers.tpl

```diff
@@ -226,3 +226,19 @@ Get ConfigMap name if existingName is defined, otherwise use default name for ge
   {{- end -}}
 {{- end }}
 
+{{/*
+Create ConfigMap checksum annotation if configMap.existingPath is defined, otherwise use default templates
+*/}}
+{{- define "opentelemetry-collector.configTemplateChecksumAnnotation" -}}
+  {{- if .Values.configMap.existingPath -}}
```

Comment:
This feature doesn't work, the path will never resolve because the custom template will not exist in the under the base path of the subchart